python/strings_udf/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,158 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+cmake_minimum_required(VERSION 3.20.1)
+
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.08/RAPIDS.cmake
```

Comment:
This file needs to be added to the `update-version.sh` script to ensure it doesn't fall out of date. See examples: https://github.com/rapidsai/cudf/blob/3099c68bf9d236c9a6d1d29b7ee5f9a4558d85cd/ci/release/update-version.sh#L37-L38